silk/datadog_checks/silk/events.py

```diff
@@ -0,0 +1,38 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+ALERT_TYPES = {
+    "INFO": "info",
+    "ERROR": "error",
+    # may have more?
```

Comment:
check the silk events page, there's 4 types AFAICT

Reply:
(info, warning, error, critical)